tests/test-wporg-two-factor.php

```diff
@@ -229,7 +229,7 @@ public function test_set_primary_provider_for_user() {
 		$totp_provider->set_user_totp_key( self::$regular_user->ID, Two_Factor_Totp::generate_key() );
 		$enabled = Two_Factor_Core::enable_provider_for_user( self::$regular_user->ID, 'Two_Factor_Totp' );
 
-		$expected = 'Two_Factor_Totp';
+		$expected = 'WordPressdotorg\Two_Factor\Encrypted_Totp_Provider';
 		$actual   = get_class( Two_Factor_Core::get_primary_provider_for_user( self::$regular_user->ID ) );
 		$this->assertTrue( $enabled );
 		$this->assertSame( $expected, $actual );
```

Comment:
This test can probably change to both check the class, but also check it's Totp as expected, as that's what we set as active above.

```suggestion
		$this->assertTrue( $enabled );

		$provider = Two_Factor_Core::get_primary_provider_for_user( self::$regular_user->ID );

		$expected_class = 'WordPressdotorg\Two_Factor\Encrypted_Totp_Provider';
		$actual_class   = get_class( $provider );
		$this->assertSame( $expected_class, $actual_class );

		$expected_key = 'Two_Factor_Totp';
		$actual_key   = $provider->get_key();
		$this->assertSame( $expected_key, $provider->get_key() );
```

Reply:
Good point, I was primarily fixing the tests before testing - makes sense to cover both.